ttnn/cpp/ttnn/operations/eltwise/ternary_backward/device/ternary_backward_op.cpp

```diff
@@ -136,7 +148,7 @@ std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tens
     }
 }
 
-std::function<std::vector<std::optional<ttnn::Tensor>>(uint8_t , const Tensor&, const Tensor&, const Tensor&, const Tensor&, const MemoryConfig&, const std::vector<bool>&, std::optional<Tensor>, std::optional<Tensor>)> get_function_type_opt(TernaryBackwardOpType OpType){
+std::function<std::vector<std::optional<Tensor>>(uint8_t , const Tensor&, const Tensor&, const Tensor&, const Tensor&, const MemoryConfig&, const std::vector<bool>&, std::optional<Tensor>, std::optional<Tensor>)> TernaryBackwardFunction::get_function_type_opt(TernaryBackwardOpType OpType){
```

Comment:
and here you should use ReturnFunction type defined above which will increase readability of your code.